feature-staking-impl/src/main/java/io/novafoundation/nova/feature_staking_impl/data/validators/NovaValidatorsApi.kt

```diff
@@ -0,0 +1,9 @@
+package io.novafoundation.nova.feature_staking_impl.data.validators
+
+import retrofit2.http.GET
+
+interface NovaValidatorsApi {
+
+    @GET("https://raw.githubusercontent.com/novasamatech/nova-utils/master/staking/nova_validators.json")
```

Comment:
Probably we also need a separate config for dev in case we want to check how recommended validators list will look like without risking users can select those validators